### DISCUSSION TITLE
Netbox installation on Rasp Pi

### DISCUSSION THREAD

**Discussion by Cronin777** [tribp/netbox#3] [Q&A]
Hi there, 
Relatively new to netbox and rasp pis. I picked up a raspberry pi to help out a company I work for in the off season to help them with network documentation. I followed the steps to install netbox and got a strange error message at step four after running the command:

sudo docker-compose up -d

The error message is
'Error response from daemon: failed to create shim task:OCI runtime create failed: runc create failed: unable to start container process: error during container init: error mounting "/etc/passwd" to rootfs at "/etc/passwd": mount /etc/passwd:/etc/passwd (via /proc/self/fd/6), flags: 0x5001: no such file or directory: unknown'

I feel like it's something silly I'm missing and would love some help. Thank you in advance.

**Comment by tribp** [comment]
Hi, I suppose that the directory '/etc/passwd' is missing on your RPI

You could try to make it: type following command into the cli on your pi

`sudo mkdir -p  /etc/passwd`

**Reply by Cronin777** [reply]
Thank you I'll try that today. There's a few things I may be doing wrong ad well. I have Xubuntu installed instead of raspbian, I failed to read that I needed to have docker installed first, and I was looking at the architecture and have not integrated a an external SSD. Sorry about the blunders, brand new to Linux and rasp pis lol.

**Reply by Cronin777** [reply]
Here's an update: 
I wiped the raspberry pi and installed bullseye, installed docker first, and followed the installation instructions again
Instead of running "sudo docker-compose -d"
I used the command "sudo docker compose -d"
I kept getting command errors with "docker-compose"

Everything seemed to be running correctly up until I got the following error:
![2023-06-03-093827_1920x1080_scrot](https://github.com/tribp/netbox/assets/122241140/10f6a94d-4ab5-45b4-a07a-e9e55650ef22)

"ERROR [netbox 2/3] RUN apk update"

**Reply by tribp** [reply]
I suppose your system misses "docker-compose" and you will need to install it.
see https://www.diyenjoying.com/2022/05/12/install-docker-compose-onto-raspberry-os/

Greetings

**Reply by Cronin777** [reply]
Thanks for your help. I feel like I'm getting closer. I can use the "docker-compose" command now but I have the following error: 
"=> ERROR [2/3] RUN apk update                                             0.9s
------
 > [2/3] RUN apk update:
------
failed to solve: process "/bin/sh -c apk update" did not complete successfully: exit code: 159"
![2023-06-03-102029_1920x1080_scrot](https://github.com/tribp/netbox/assets/122241140/db212345-0e76-4258-bfee-80a7ceba6e50)
